code/datums/station_traits/_station_trait.dm

```diff
@@ -16,7 +16,7 @@ GLOBAL_LIST_EMPTY(lobby_station_traits)
 	///Whether this trait is always enabled; generally used for debugging
 	var/force = FALSE
 	///Does this trait show in the centcom report?
-	var/show_in_report = FALSE
+	var/show_in_report = TRUE
```

Comment:
Maybe we should just replace this boolean with checking if `report_message` has any contents.